pineappl_capi/src/lib.rs

```diff
@@ -847,7 +847,7 @@ pub unsafe extern "C" fn pineappl_grid_read(filename: *const c_char) -> Box<Grid
 ///
 /// # Safety
 ///
-/// The parameter `grid` must be valid `Grid` object created by either `pineappl_grid_new` or
+/// The `grid` must be valid `Grid` object created by either `pineappl_grid_new` or
```

Comment:
```suggestion
/// The parameter `grid` must be valid `Grid` object created by either `pineappl_grid_new` or
```
No need to just delete it here